static/js/search_suggestion.js

```diff
@@ -252,7 +270,11 @@ function get_person_suggestions(people_getter, last, operators, autocomplete_ope
 
     const objs = persons.map((person) => {
         const name = highlight_person(person);
-        const description = prefix + " " + name;
+        const description = i18n.t("__prefix__ __name__", {
```

Comment:
hmm. All the translators will see in the web UI would be the string `__prefix__ __name__ `. 

The value of `__prefix__` can be "private messages with" for example in web app run time. But that is not visible to translators. All they can see is `__prefix__ __name__` in the dashboard and they will have no idea what to do with that.

Also making `Filter.operator_to_prefix` return a semi translated version(eg i18n.t("private messages with");) is not useful. Translators needs full sentence to translate the message. We can't just append two parts of the sentences into one.